assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -94,9 +95,22 @@ class AnswerForm extends React.PureComponent<*, AnswerFormProps, AnswerFormState
         };
         displayAlert('success', I18n.t('loading.wait'));
         createPost({ variables: variables })
-          .then(() => {
-            refetchIdea().then(() => {
-              this.setState({ body: null }, hideAnswerForm);
+          .then((res) => {
+            const postId = res.data.createPost.post.id;
+            this.setState({ submitting: false, body: null }, () => {
+              hideAnswerForm();
+              // Execute refetchIdea after the setState otherwise we can get a warning setSate called on unmounted component.
+              refetchIdea().then(() => {
+                // The thread may have moved because it became the latest active, we need to scroll to the created post.
+                // Arbitrary wait 200ms to be sure the post is rendered.
+                setTimeout(() => {
+                  const createdPostElement = document.getElementById(postId);
+                  if (createdPostElement) {
+                    const elmOffset = getDomElementOffset(createdPostElement).top - 200;
```

Comment:
What is with the other arbitrary 200px shift?

Reply:
Found by trial and error. I want the line expand/collapse + above the post to appear. Maybe I can do a more complicated stuff and get the height of some element instead of hard coding 200px here. I'll look into it.